src/locales/fi.json

```diff
@@ -675,6 +675,10 @@
     "berths": {
       "title": "Venepaikka",
       "description": "Perushinta. Hinnat sisältävät alv 24 %.",
+      "defaultProducts": "Venepaikat",
+      "vasikkasaari": "Vasikkasaaren venesatama",
+      "dinghy": "Jollapaikat",
+      "trailer": "Traieripaikat",
```

Comment:
Minor typo, missing the letter L from traileri

Reply:
Good catch, fixed.